src/daemon/utils_cache.c

```diff
@@ -42,7 +42,7 @@
 
 typedef struct cache_entry_s {
   char name[6 * DATA_MAX_NAME_LEN];
-  distribution_t *values_distribution;
+  distribution_t *distribution_increase;
```

Comment:
What is the meaning of this attribute?

Reply:
distribution_increase shows us how the distribution has been updated since last time when we saw it. Using it will able to calculate rate. This change was just to rename values_distribution as it was unclear what exactly it means